script/prep-travis-release.sh

```diff
@@ -92,7 +99,7 @@ BASENAME="rack"
 RACKBUILD="${BASENAME}${SUFFIX}"
 
 COMMIT=$(git rev-parse --verify HEAD)
-sed -i "s/var Commit =.*/var Commit = \"$COMMIT\"/" util/util.go
+sed -i "s/var Commit =.*/var Commit = \"$COMMIT\"/" util/commit.go
```

Comment:
Down here you'll want to change `var Version` as well so it's built into the binary.